hoomd/hpmc/ComputeSDF.h

```diff
@@ -486,6 +486,8 @@ template<class Shape> void ComputeSDF<Shape>::countHistogramLinearSearch(uint64_
     const LongReal min_core_radius = m_mc->getMinCoreDiameter() * LongReal(0.5);
     const auto& pair_energy_search_radius = m_mc->getPairEnergySearchRadius();
 
+    Scalar kT = m_mc->getTimestepkT(timestep); // use const?
```

Comment:
```suggestion
    const Scalar kT = (*m_mc->getKT())(timestep);
```
I would prefer that we call `operator()` on `Variant` directly.